### PR TITLE
[FIXED JENKINS-44665] - "Select all" button inverts all permissions in the Matrix Auth Security settings

### PR DESCRIPTION
**Pull request by ksenia-nenasheva**
See [JENKINS-44665](https://issues.jenkins-ci.org/browse/JENKINS-44665).

Details:
The old button did inversion for all permissions in the Matrix Auth Security settings:

- Delete old button
- Add new "Select all" and "Unselect all" buttons with new icons.

### CODE REVIEW

**Comment by ksenia-nenasheva** [comment]
@jenkinsci/code-reviewers

**Comment by daniel-beck** [comment]
Why add both png and gif?

**Comment by ksenia-nenasheva** [comment]
@daniel-beck Done.
While my PR has been merged, PR https://github.com/jenkinsci/jenkins/pull/1602 can be closed ([issue #26544](https://issues.jenkins-ci.org/browse/JENKINS-26544))

File: src/main/resources/hudson/security/GlobalMatrixAuthorizationStrategy/config.jelly [hunk 1]
```diff
@@ -152,12 +155,23 @@ THE SOFTWARE.
           }
           e = null; <!-- avoid memory leak -->
         });
-      Behaviour.specify("#${strategyid} TD.stop A.toggleall", 'GlobalMatrixAuthorizationStrategy', 0, function(e) {
+      Behaviour.specify("#${strategyid} TD.stop A.selectall", 'GlobalMatrixAuthorizationStrategy', 0, function(e) {
+          e.onclick = function() {
+            var tr = findAncestor(this,"TR");
+            var inputs = tr.getElementsByTagName("INPUT");
+            for(var i=0; i&lt;inputs.length; i++){
```
Comment (jglick) [inline src/main/resources/hudson/security/GlobalMatrixAuthorizationStrategy/config.jelly]: Would be nice to share the common code in a separate function.

**Pull request merged**